src/test/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilderTest.java

```diff
@@ -324,9 +327,8 @@ public void testOverridesFromYaml() throws Exception {
     }
 
     /**
-     * This is counter intuitive, the yaml contents are ignored because the parent fields are merged first with the
-     * child ones. Then the fields override what is defined in the yaml, so in effect the parent resource limits and
-     * requests are used.
+     * Overrride is in this order:
+     * parent (UI) > template (UI) > yaml
```

Comment:
Not clear to me what this is saying. What is `>` here?

Reply:
is overridden

Reply:
Hmm, for that I would expect `<` perhaps. Either way, needs to be expressed in the comment itself.